packages/preferences-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.1]
```

Comment:
For future reference, we should use minor bumps for new features. Not significant enough to warrant correcting now that this is merged though imo.